app/services/trials.js

```diff
@@ -86,6 +86,8 @@ function _processData(trials) {
 
         // Computed values here
         daysAfterCompletion: today.diff(trial.completion_date, 'days'),
+        daysBeforeCompletion: _cleanDate(trial.completion_date)
+          .diff(today, 'days'),
```

Comment:
And it supports the case where `trial.completion_date == '-'` or other invalid date?